src/components/NewSnippet.jsx

```diff
@@ -1,10 +1,60 @@
 import React from 'react';
+import { Scrollbars } from 'react-custom-scrollbars';
+import { Controlled as CodeMirror } from 'react-codemirror2';
 
+import 'codemirror/lib/codemirror.css';
+
+import Title from './common/Title';
+import Input from './common/Input';
+
+import '../styles/NewSnippet.styl';
 
 const NewSnippet = () => (
-  <div>
-    New Snippet page
-  </div>
+  [
+    <Title title="New snippet" key="title" />,
+    <div className="new-snippet" key="new-snippet">
+      <div className="new-snippet-code-wrapper">
+        <div className="new-snippet-code-header">
+          <Input placeholder="Title" />
+          <Input placeholder="Tags (separate tags by comma)" />
+        </div>
+        <div className="new-snippet-code">
+          <CodeMirror
+            value="console.log('All heil XSnippet')"
+            options={{ lineNumbers: false }}
+          />
+          <div className="new-snippet-code-bottom-bar">
+            <button>POST</button>
+          </div>
+        </div>
+      </div>
+      <div className="new-snippet-lang-wrapper">
+        <div className="new-snippet-lang-header">
+          <Input placeholder="Type to search..." />
+        </div>
+        <div className="new-snippet-lang-list-wrapper">
+          <Scrollbars>
+            <ul className="new-snippet-lang-list">
+              <li className="new-snippet-lang-item">Text1</li>
```

Comment:
I'll add a new route to the API to retrieve this.

Reply:
Great, but it's not urgent. I just thought that I will create all markup with stabbed data at first  and only then divide into small component and add logic